config/fcos/v1_5_exp/translate_test.go

```diff
@@ -1417,3 +1417,59 @@ func TestTranslateBootDevice(t *testing.T) {
 		assert.NoError(t, translations.DebugVerifyCoverage(actual), "#%d: incomplete TranslationSet coverage", i)
 	}
 }
+
+// TestTranslateExtensions tests translating the Butane config extensions section.
+func TestTranslateExtensions(t *testing.T) {
+	tests := []struct {
+		in         Config
+		out        types.Config
+		exceptions []translate.Translation
+		report     report.Report
+	}{
+		// config with two extensions/packages
+		{
+			Config{
+				Extensions: []string{"strace", "zsh"},
+			},
+			types.Config{
+				Ignition: types.Ignition{
+					Version: "3.4.0-experimental",
+				},
+				Storage: types.Storage{
+					Files: []types.File{
+						{
+							Node: types.Node{
+								Path: "/etc/rpm-ostree/origin.d/extensions-e2ecf66.yaml",
+							},
+							FileEmbedded1: types.FileEmbedded1{
+								Contents: types.Resource{
+									Source: util.StrToPtr("data:;base64,IyBHZW5lcmF0ZWQgYnkgQnV0YW5lCgpwYWNrYWdlczoKICAgIC0gc3RyYWNlCiAgICAtIHpzaAo="),
+								},
+								Mode: util.IntToPtr(420),
+							},
+						},
+					},
+				},
+			},
+			[]translate.Translation{
+				{path.New("yaml", "version"), path.New("json", "ignition", "version")},
+				{path.New("yaml", "extensions"), path.New("json", "storage")},
+				{path.New("yaml", "extensions"), path.New("json", "storage", "files")},
+				{path.New("yaml", "extensions"), path.New("json", "storage", "files", 0)},
+				{path.New("yaml", "extensions"), path.New("json", "storage", "files", 0, "path")},
+				{path.New("yaml", "extensions"), path.New("json", "storage", "files", 0, "mode")},
+				{path.New("yaml", "extensions"), path.New("json", "storage", "files", 0, "contents")},
+				{path.New("yaml", "extensions"), path.New("json", "storage", "files", 0, "contents", "source")},
+			},
+			report.Report{},
+		},
+	}
+
+	for i, test := range tests {
```

Comment:
e83e1c99d8e45b7f8e0add5dffaef9751d34370f introduced a style change for these loops.  Could you apply the update when you next respin?

Reply:
done